CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/MapViewTopComponent.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.mapview2;
+
+import au.gov.asd.tac.constellation.graph.Graph;
+import au.gov.asd.tac.constellation.plugins.PluginExecution;
+import au.gov.asd.tac.constellation.views.JavaFxTopComponent;
+import au.gov.asd.tac.constellation.views.mapview.providers.MapProvider;
+import au.gov.asd.tac.constellation.views.mapview2.markers.AbstractMarker;
+import au.gov.asd.tac.constellation.views.mapview2.plugins.ExtractCoordsFromGraphPlugin;
+import java.awt.Component;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javafx.application.Platform;
+import javafx.embed.swing.JFXPanel;
+import javafx.scene.control.ScrollPane;
+import javax.swing.SwingUtilities;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle;
+import org.openide.windows.TopComponent;
+
+/**
+ * Top component for the MapView
+ *
+ * @author altair1673
+ */
+@TopComponent.Description(
+        preferredID = "MapViewTopComponent2",
+        iconBase = "au/gov/asd/tac/constellation/views/mapview/resources/treasure-map.png",
+        persistenceType = TopComponent.PERSISTENCE_ALWAYS
+)
+@TopComponent.Registration(
+        mode = "explorer",
+        openAtStartup = false
+)
+@ActionID(
+        category = "Window",
+        id = "au.gov.asd.tac.constellation.views.mapview2.MapViewTopComponent"
+)
+@ActionReferences({
+    @ActionReference(path = "Menu/Experimental/Views", position = 4000),
+    @ActionReference(path = "Shortcuts", name = "CA-M")
```

Comment:
Our keyboard shortcuts for views (with the exception of find) have been ctrl-shift and ideally I think it would be good to keep that consistent. Perhaps in this case, if there is no other logical option, perhaps we just leave the shortcut out for now and when it becomes the main view it can inherit CS-M